sdk/personalizer/Azure.AI.Personalizer/src/Models/PersonalizerClient.cs

```diff
@@ -70,13 +72,16 @@ public PersonalizerClient(Uri endpoint, TokenCredential credential, Personalizer
         /// <param name="endpoint"> Supported Cognitive Services endpoint. </param>
         /// <param name="credential"> A credential used to authenticate to an Azure Service. </param>
         /// <param name="isLocalInference"> A flag to determine whether to use local inference. </param>
+        /// <param name="interactionsSubsampleRate"> Percentage from (0,1] determines how much percentage of interaction events to consider </param>
+        /// <param name="observationsSubSampleRate"> Percentage from (0,1] determines how much percentage of observation events to consider </param>
         /// <param name="options"> The options for configuring the client. </param>
-        public PersonalizerClient(Uri endpoint, TokenCredential credential, bool isLocalInference, PersonalizerClientOptions options = null) :
+        public PersonalizerClient(Uri endpoint, TokenCredential credential, bool isLocalInference, float interactionsSubsampleRate = 1.0f, float observationsSubSampleRate = 100, PersonalizerClientOptions options = null) :
```

Comment:
Please only have one subsampling rate. Note, you have two different ranges. One is 1s based and the other 100s based.

Reply:
update to one subsampling rate